client/src/app/site/motions/components/category-list/category-list.component.ts

```diff
@@ -188,8 +202,19 @@ export class CategoryListComponent extends BaseComponent implements OnInit {
      * @param category target
      */
     public motionsInCategory(category: Category): Array<Motion> {
-        const motList = this.repo.getMotionsOfCategory(category);
-        motList.sort((motion1, motion2) => (motion1 > motion2 ? 1 : -1));
-        return motList;
+        const motions = this.repo.getMotionsOfCategory(category);
+        motions.sort((motion1, motion2) => (motion1 > motion2 ? 1 : -1));
+        return motions;
+    }
+
+    /**
+     * Is executed when a mat-extension-panel is closed
+     * @param viewCategory the category in the panel
+     */
+    public panelClosed(viewCategory: ViewCategory): void {
+        this.openId = null;
+        if (this.editId) {
+            this.onSaveButton(viewCategory);
```

Comment:
why should something be saved just because you close a panel?
If anything, changes should be dismissed.

Reply:
We discussed this, as I implemented this feature. It was wanted behaviour.